pkg/webhooks/validator/validator.go

```diff
@@ -286,8 +292,11 @@ func (wh WebhookHandler) updateOperatorCr(ctx context.Context, hc *v1beta1.Hyper
 
 	switch existing := exists.(type) {
 	case *kubevirtcorev1.KubeVirt:
-		required, err := operands.NewKubeVirt(hc)
-		if err != nil {
+		required, err := operands.NewKubeVirt(ctx, wh.cli, hc)
+
+		if errT, ok := err.(*operands.RateTuningError); ok {
+			wh.logger.Error(errT, "failed to set the rate limit")
```

Comment:
Nit: can drop `errT`. logger.Error will work with the existing `err` variable.

Reply:
Done.